homeassistant/components/mqtt/__init__.py

```diff
@@ -250,6 +251,63 @@
 SubscribePayloadType = Union[str, bytes]  # Only bytes if encoding is None
 
 
+class MqttCommandTemplate:
+    """Class for rendering MQTT payload with command templates."""
+
+    def __init__(
+        self,
+        command_template: template.Template | None,
+        hass: HomeAssistant,
+        variables: template.TemplateVarsType = None,
+    ) -> None:
+        """Instantiate a command template."""
+        self._attr_command_template = command_template
+        if command_template is None:
+            return
+
+        command_template.hass = hass
+        self.variables = variables
+
+    @callback
+    def async_render(
+        self,
+        value: PublishPayloadType = None,
+        variables: template.TemplateVarsType = None,
+    ) -> PublishPayloadType:
+        """Render or convert the command template with given value or variables."""
+        if self._attr_command_template is None:
+            return MqttCommandTemplate._convert_outgoing_payload(value)
```

Comment:
Is this wanted when there is no template, or should we just do:
```py
return value
```

Reply:
You are right, we should just return here.